doc/modules/ROOT/pages/vcdj.adoc

```diff
@@ -562,6 +568,37 @@ which the cue point was saved the value is `0000` and the CDJ displays
 “`00.0 Bars`”. On the next beat, the value becomes determined by the
 next cue point (if any) in the track.
 
+Byte `b7`, labeled _Mp_, seems to indicate local media presence status
+for players with multiple media slots. For players with only one slot,
+this value remains `0`.
+
+[[media-presence-flag-bits]]
+.Media presence flag bits.
+[bytefield]
+----
+(def boxes-per-row 8)
+(def box-width 70)
+(def left-margin 1)
+(draw-column-headers {:labels (str/split "7,6,5,4,3,2,1,0" #",")})
+(draw-box (hex-text 0 1))
+(draw-box (hex-text 0 1))
+(draw-box (hex-text 0 1))
+(draw-box (hex-text 0 1))
+(draw-box (hex-text 0 1))
+(draw-box (hex-text 0 1))
+(draw-box "USB")
+(draw-box "SD")
+----
+
+Byte `b8`, labeled _Ue_, indicates if USB media has been ejected.
+If media is inserted, this value returns to `0`.
+
+Byte `b9`, labeled _Se_, indicates if SD media has been ejected.
+If media is inserted, this value returns to `0`.
```

Comment:
What are the values of `b8` and `b9` after power on, but before any media has been inserted? Are they still `0`, or do they start out as `1`? If the latter, then we might want to describe them as “media missing” rather than “media ejected”? Either way, it seems redundant with the values in the media presence byte, which in turn seem redundant with the values in Ul (`6f`) and Sl (`73`). But these packets are weird, we already know that.

Reply:
After power on the values are curiously `0` even if media is missing. Inserting media keeps the value `0`, then removing it flips it to `1`. Indeed it seems that media state can be inferred from the Ul and Sl flags.

Reply:
Curious indeed! Thanks for the thorough research. Sometimes I get the feeling these packets are just created by sending a snapshot of a section of memory where data structures used by the player itself live. 😆